plugins/views/frontend/public/javascripts/countly.models.js

```diff
@@ -448,18 +448,42 @@
         return getRange().indexOf(value);
     };
 
+    countlyViews.renameViews = function(data, callback) {
+        $.ajax({
+            type: "POST",
+            url: countlyCommon.API_PARTS.data.w + '/views',
+            data: {
+                "app_id": countlyCommon.ACTIVE_APP_ID,
+                "method": "rename_views",
+                "data": JSON.stringify(data)
+            },
+            dataType: "json",
+            success: function(json) {
+                callback && callback(json);
+            },
+            error: function() {
+                callback && callback(false);
+            }
+        });
+    };
     countlyViews.deleteView = function(view, callback) {
         return $.ajax({
             type: "POST",
-            url: countlyCommon.API_PARTS.data.w + '/delete_view',
+            url: countlyCommon.API_PARTS.data.w + '/views',
             data: {
                 "app_id": countlyCommon.ACTIVE_APP_ID,
                 "method": "delete_view",
                 "view_id": view
             },
             dataType: "json",
             success: function(json) {
+                if (json && json.result) {
+                    json = json.result;
+                }
                 callback && callback(json);
+            },
+            error: function() {
+                callback && callback(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5139130)